grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -51,14 +149,615 @@ class ApiGetOpenApiDescriptionHandler(api_call_handler_base.ApiCallHandler):
   args_type = None
   result_type = ApiGetOpenApiDescriptionResult
 
+  def __init__(self, router: Any) -> None:
+    # TODO(hanuszczak): Break dependency cycle between this module and
+    # `api_call_router.py` and then use ApiCallRouter as self.router's and the
+    # argument's type.
+    self.router: Any = router
+    # The main OpenAPI description object.
+    self.openapi_obj: Optional[Dict[str, Union[str, List, Dict]]] = None
+    self.openapi_obj_json: Optional[str] = None
+    self.schema_objs: Optional[Dict[str, Schema]] = None
+
+  def _NormalizePathComponent(self, component: str) -> str:
+    """Normalize the given path component to be used in a valid OpenAPI path."""
+    if len(component) > 0 and component[0] == '<' and component[-1] == '>':
```

Comment:
I think `component.startswith("<") and component.endswith(">")` reads more naturally. Also, use double quotes for string literals (I suspect that you used single ones because of single characters, but the unfortunate reality is that there is no `char` type in Python).

Reply:
Thank you for pointing this out! I changed all `'`s to `"`s where possible. Probably my subconscious was fixed on C/C++ chars and on SQL syntax :satisfied:.